<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resources</class>
 <widget class="QDialog" name="Resources">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resources</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">.QFrame
{
	border: 1px solid palette(text);
}

.QFrame&gt;QLabel
{
	background-color: palette(text);
	color: palette(window);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabMusic">
      <attribute name="title">
       <string>Music</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="lstMusic">
         <property name="styleSheet">
          <string notr="true">QListWidget::indicator
{
	image: url(:/icons/play.png);
}

QListWidget::indicator:checked
{
	image: url(:/icons/stop.png);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Volume:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sldMusicVolume">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblMusicVolume">
            <property name="text">
             <string>100%</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSFX">
      <attribute name="title">
       <string>SFX</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="lstSFX">
         <property name="styleSheet">
          <string notr="true">QListWidget::indicator
{
	image: url(:/icons/play.png);
}

QListWidget::indicator:checked
{
	image: url(:/icons/stop.png);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Volume:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sldSFXVolume">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblSFXVolume">
            <property name="text">
             <string>100%</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBackdrops">
      <attribute name="title">
       <string>Backdrops</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="lstBackdrops">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>128</height>
          </size>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <item>
          <property name="text">
           <string>Cavern</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../resources/backdrops/cavern.png</normaloff>../../resources/backdrops/cavern.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Desert</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../resources/backdrops/desert.png</normaloff>../../resources/backdrops/desert.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Forest</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../resources/backdrops/forest.png</normaloff>../../resources/backdrops/forest.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Field</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../resources/backdrops/field.png</normaloff>../../resources/backdrops/field.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mountain</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../resources/backdrops/mountain.png</normaloff>../../resources/backdrops/mountain.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mountain 2</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../resources/backdrops/mountain2.png</normaloff>../../resources/backdrops/mountain2.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ship Deck</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../resources/backdrops/shipdeck.png</normaloff>../../resources/backdrops/shipdeck.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Swamp</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../resources/backdrops/swamp.png</normaloff>../../resources/backdrops/swamp.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tribal Pit</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../resources/backdrops/tribalPit.png</normaloff>../../resources/backdrops/tribalPit.png</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>Resources</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
